src/GUniLPOracle.sol

```diff
@@ -217,6 +215,14 @@ contract GUniLPOracle {
         emit Start();
     }
 
+    function void() external auth {
+        stopped = 1;
+        delete cur;
+        delete nxt;
+        zph = 0;
+        emit Void();
+    }
+
     function step(uint256 _hop) external auth {
```

Comment:
Optionally we could add here this logic:
https://github.com/makerdao/curve-lp-oracle/blob/master/src/CurveLPOracle.sol#L145

That was to keep consistency with the old behavior when a `hop` change would affect the time of the next update. But it's also fine if we prefer to keep this PR tightly-scoped.

Reply:
I would say if we want this change, maybe better to have a separate PR that also gets merged before redeploy?

Reply:
Yes, let's evaluate that in another PR.